libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -1301,7 +1301,7 @@ open class SalesforceSDKManager protected constructor(
         @Suppress("SameParameterValue") fieldName: String
     ) = runCatching {
         Class.forName(
-            "${context.javaClass.getPackage()?.name ?: ""}.BuildConfig"
```

Comment:
Note this was the only place Salesforce SDK Manager used the context's class package name.  Everything else was already using the Context package name.